Rules/WAR/gamemode.cfg

```diff
@@ -25,7 +25,7 @@
                                          TimeToEnd.as;
                                          PostGameMapVotes.as; #RestartAfterShortPostGame.as;
                                          FanfareOnWin.as;
-                                         TeamWinBanner.as;
+                                         GameStateBanners.as;
```

Comment:
Other gamemodes shouldn't include this as it is a dependency script

Reply:
resolved